backend/src/modules/users/users.service.ts

```diff
@@ -27,15 +19,9 @@ export class UsersService {
       password: await argon2.hash(createUserDto.password),
       roleId: createUserDto.roleId,
     });
-
-    const token = this.jwtService.sign({ email: createUserDto.email });
   }
 
   async findOne(email: string) {
-    return await this.userRepository.findOne({
-      where: {
-        email: email,
-      },
-    });
+    return await this.userRepository.findUserByEmail(email);
```

Comment:
We don't need await here
https://kanby.medium.com/%D0%BF%D0%BE%D1%87%D0%B5%D0%BC%D1%83-%D0%B8%D1%81%D0%BF%D0%BE%D0%BB%D1%8C%D0%B7%D0%BE%D0%B2%D0%B0%D1%82%D1%8C-return-await-%D0%BF%D0%BB%D0%BE%D1%85%D0%B0%D1%8F-%D0%B8%D0%B4%D0%B5%D1%8F-e87b70015f0c


https://learn.javascript.ru/async-await